File: DSA/Algorithm/Mathematical/Easy/Count_Integers_With_Even_Digit_Sum/Solutions/Solve.js
/**
 * @param {number} num
 * @return {number}
 */
 var countEven = function(num) {
    let count = 0;
    for(let i = 2; i <= num; i++){
        let nums = "" + i;
        let check = 0;
        for(let j = 0; j < nums.length; j++){
            check += +nums[j];
        }
        if(check % 2 === 0) count++;
    }
    return count;
};

File: DSA/Data_Structure/Arrays/Easy/Two_Sum/Solutions/solve.py
def twoSum(nums, target):
    obj = {}
    for index, num in enumerate(nums):
        check = str(target - num)
        if check in obj:
            return [obj[check], index]
        else:
            obj[str(num)] = index
    return

def main():
    print(twoSum([2,7,11,15], 9))

main()